catalyst-gateway/event-db/migrations/V2__signed_documents.sql

```diff
@@ -10,11 +10,11 @@
 
 -- -------------------------------------------------------------------------------------------------
 
--- Signed Documents Storage Repository defintion.
+-- Signed Documents Storage Repository definition.
 CREATE TABLE IF NOT EXISTS signed_docs (
   id UUID NOT NULL, -- UUID v7
   ver UUID NOT NULL, -- UUID v7
-  type UUID NOT NULL, -- UUID v4
+  type UUID [] NOT NULL, -- List of UUID v4
```

Comment:
NAK.  Can't change schema like this.  Requires a `V3_update_signed_docs.sql` which performs a table modification of this one column.  And it needs to be able to migrate the data.